students/11th/yejikang/js/main.js

```diff
@@ -0,0 +1,36 @@
+const commentContainer = document.querySelector('.commentContainer');
+const commentForm = document.querySelector('.commentForm');
+const commentBtn = commentForm.querySelector('button');
+const inputComment = document.querySelector('.inputComment');
+
+const userId = "yejiii_95";  
+
+function addComment() {
+    if(inputComment.value.trim('').length===0) return;
```

Comment:
```suggestion
    if(!inputComment.value.trim('')) return;
```
1. 자바스크립트에서 0은 falsy한 값입니다. 그 자체로 true, false 값을 갖는다는 뜻입니다.
2. 부정연산자(!)를 사용하면 불리언 값을 뒤집어줄 수 있습니다. if문의 조건식이 true가 되면 실행되므로 이 경우 위와 같이 입력하면 빈칸인 경우를 뜻하게 됩니다.

Reply:
수정했습니다.

Reply:
수정했습니다 감사합니다!